client/src/app/authentication/auth.service.ts

```diff
@@ -1,138 +1,90 @@
-    concatMap((client: Auth0Client) => from(client.isAuthenticated())),
-    tap(res => this.loggedIn = res)
-  );
+     domain: environment.AUTH_DOMAIN,
+     client_id: environment.AUTH_CLIENT_ID,
+     redirect_uri: environment.BASE_URL,
+     audience: environment.AUTH_API_DOMAIN
+   })).pipe(shareReplay(1), catchError(err => throwError(err)));
+
+  isAuthenticated$ = this.auth0Client$.pipe(concatMap((client: Auth0Client) =>
+    from(client.isAuthenticated())),
+    tap(res => this.loggedIn = res));
+
   handleRedirectCallback$ = this.auth0Client$.pipe(
-    concatMap((client: Auth0Client) => from(client.handleRedirectCallback()))
-  );
-  // Create subject and public observable of user profile data
+    concatMap((client: Auth0Client) => from(client.handleRedirectCallback())));
+
   private userProfileSubject$ = new BehaviorSubject<any>(null);
   userProfile$ = this.userProfileSubject$.asObservable();
-  // Create a local property for login status
   loggedIn: boolean = null;
 
-  constructor(private router: Router) {
-    // On initial load, check authentication state with authorization server
-    // Set up local auth streams if user is already authenticated
+  constructor(public router: Router) {
     this.localAuthSetup();
-    // Handle redirect from Auth0 login
     this.handleAuthCallback();
   }
 
-  // When calling, options can be passed if desired
-  // https://auth0.github.io/auth0-spa-js/classes/auth0client.html#getuser
+  getTokenSilently$(options?): Observable<string> {
+    return this.auth0Client$.pipe(concatMap((client: Auth0Client) =>
+      from(client.getTokenSilently(options))));
+  }
+
   getUser$(options?): Observable<any> {
     return this.auth0Client$.pipe(
       concatMap((client: Auth0Client) => from(client.getUser(options))),
       tap(user => this.userProfileSubject$.next(user))
     );
   }
 
-  getTokenSilently$(options?): Observable<string> {
-    return this.auth0Client$.pipe(
-      concatMap((client: Auth0Client) => from(client.getTokenSilently(options)))
-    );
-  }
-
   private localAuthSetup() {
-    // This should only be called on app initialization
-    // Set up local authentication streams
     const checkAuth$ = this.isAuthenticated$.pipe(
       concatMap((loggedIn: boolean) => {
         if (loggedIn) {
-          // If authenticated, get user and set in app
-          // NOTE: you could pass options here if needed
           return this.getUser$();
         }
-        // If not authenticated, return stream that emits 'false'
         return of(loggedIn);
       })
     );
     checkAuth$.subscribe();
   }
 
-  login(redirectPath: string = '/') {
-    // A desired redirect path can be passed to login method
-    // (e.g., from a route guard)
-    // Ensure Auth0 client instance exists
-    this.auth0Client$.subscribe((client: Auth0Client) => {
-      // Call method to log in
-      client.loginWithRedirect({
+  login(redirectPath: string = '/home/dashboard'): Observable<void> {
```

Comment:
This doesn't look right to me. I believe the string should just be '/' for our project

Reply:
You're right, I missed that one.